### PR TITLE
Cleaned up some docs

### PR DESCRIPTION
**Pull request by GeorgeSittas**
I removed the `expressions.md` document in favor of improving / adding some docstrings in `expressions.py`.

I think that the existing documentation (docstrings) on the helper functions is not too bad, but it may not be easy enough to find looking at the docs. Any suggestion on how we can improve this?

Also, it feels like there are several commenting styles in the codebase, so it can feel a bit inconsistent. Should we establish a style and stick with it from now on?

### CODE REVIEW

**Review by tobymao — APPROVED** [review]

File: sqlglot/dataframe/README.md [hunk 1]
```diff
@@ -7,24 +7,24 @@ Currently many of the common operations are covered and more functionality will
...
-      * Ex: `StructType([StructField('cola', StringType()), StructField('colb', IntegerType())])`
-    * A string of names and types similar to what is supported in `createDataFrame`
-      * Ex: `cola: STRING, colb: INT`
-    * [Not Recommended] A list of string column names without type
-      * Ex: ['cola', 'colb']
-      * The lack of types may limit functionality in future releases
-  * See [Registering Custom Schema](#registering-custom-schema-class) for information on how to skip this step if the information is stored externally
-* Add `.sql(pretty=True)` to your final DataFrame command to return a list of sql statements to run that command
+    * Dictionary where the keys are column names and values are string of the Spark SQL type name.
+      * Ex: `{'cola': 'string', 'colb': 'int'}`.
```
Comment (barakalon) [inline sqlglot/dataframe/README.md:15]: Things that aren't sentences don't periods

File: sqlglot/__init__.py [hunk 2]
```diff
@@ -35,8 +36,10 @@
 __version__ = "10.5.6"
 
 pretty = False
+"""Whether to format generated SQL by default."""
```
Comment (barakalon) [inline sqlglot/__init__.py:39]: is this a pdocs thing? Docstrings under a variable?

**Review by barakalon — APPROVED** [review]

File: sqlglot/dataframe/README.md (hunk 1 above)
Comment (GeorgeSittas) [reply sqlglot/dataframe/README.md:15]: Wasn't aware, thanks for clarifying.

File: sqlglot/__init__.py (hunk 2 above)
Comment (GeorgeSittas) [reply sqlglot/__init__.py:39]: Yep, it makes the variable appear in the API documentation:

Screenshot 2023-01-26 at 6 53 16 PM

**Pull request merged**